src/shelly/commands/npmTrustedPublishing.ts

```
@@ -0,0 +1,789 @@
+  private async simulateWorkflowUpdate(
+    workflowPath: string
+  ): Promise<{ changes: string[] }> {
+    const workflow = await this.npmService.analyzeWorkflow(workflowPath);
+    const changes: string[] = [];
+
+    if (!workflow.hasIdTokenPermission) {
+      changes.push('Add id-token: write permission');
+    }
+
+    if (
+      workflow.publishCommands.length > 0 &&
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)